recipes/aeppl/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.6" %}
+
+package:
+  name: aeppl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/aeppl/aeppl-{{ version }}.tar.gz
+  sha256: b0162e21c86ace1c698d5104aac003c994528ce46cd50f643b5a86ed6852dc2d
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Does this package really require compilers at *run* time (i.e. while the user is making use of the package)?

Reply:
No, this particular package doesn't.

Reply:
Well, not if those dependencies are picked up via the Aesara dependency.  Anyway, I've removed them.